benchmark/Streamly/Benchmark/Data/Array.hs

```diff
@@ -184,8 +184,13 @@ defStreamSize = defaultStreamSize
 
 main :: IO ()
 main = do
+#if TASTY
+    (value, ingredients) <- parseCLIOpts defStreamSize $ bgroup "All" (allBenchmarks 0)
```

Comment:
Damn! I did not realize this `All` prefix is being added by us, I assumed tasty adds it automatically. I had to do a lot of special handling in bench.sh because of this. We should not add this, why?

Reply:
One must group all benchmarks together, but the name is arbitrary, even an empty string will do.

Reply:
"All" prefix is hard coded in **defaultMain**  method:
defaultMain :: [Benchmark] -> IO ()
defaultMain = Test.Tasty.defaultMainWithIngredients benchIngredients . testGroup "All"

Reply:
We can use `defaultMainWithIngredients` instead of `defaultMain` and use the prefix as "" (empty string)?

Reply:
Do we need to remove "All" prefix and fix bench.sh?

Reply:
If we can do that it will be great.